src/test/java/roomescape/acceptance/ReservationAcceptanceTest.java

```diff
@@ -111,6 +101,22 @@ Stream<DynamicTest> reservationPostAndDeleteTest() {
         );
     }
 
+    @DisplayName("결제가 실패하면 예약이 생성되지 않는다.")
+    @Test
+    void reservationPostWhenPaymentFail() {
+        LocalDate tomorrow = LocalDate.now().plusDays(1);
+        ReservationRequest request = new ReservationRequest(tomorrow, 1L, 1L, null, null, 1000);
+        PaymentRequest paymentRequest = new PaymentRequest(request.orderId(), request.amount(),
+                request.paymentKey());
+        BDDMockito.given(paymentService.pay(paymentRequest))
+                .willThrow(new PaymentException(
+                        HttpStatus.BAD_REQUEST, new TossExceptionResponse("EXCEPTION", "exception")));
```

Comment:
static import를 활용해봅시다!
```suggestion
        given(paymentService.pay(paymentRequest))
                .willThrow(new PaymentException(
                        HttpStatus.BAD_REQUEST, new TossExceptionResponse("EXCEPTION", "exception")));
```